packages/supabase_flutter/lib/src/supabase_auth.dart

```diff
@@ -100,23 +100,23 @@ class SupabaseAuth with WidgetsBindingObserver {
 
   /// Recover/refresh session if it's available
   /// e.g. called on a splash screen when the app starts.
-  Future<bool> _recoverSupabaseSession() async {
+  Future<void> _recoverSupabaseSession() async {
```

Comment:
Changing the return type is not related to fixing the issue for this PR, but since the returned value was not used anywhere, I thought it would reduce any confusions in the future.